examples/transfer-opts/index.mjs

```diff
@@ -0,0 +1,28 @@
+import {loadStdlib} from '@reach-sh/stdlib';
+const stdlib = loadStdlib(process.env);
+
+const startingBalance = stdlib.parseCurrency(100);
+
+const [ accAlice, accBob, accCharlie ] =
+  await stdlib.newTestAccounts(3, startingBalance);
+
+if (stdlib.connector === 'ALGO') {
```

Comment:
It should still work on ETH

Reply:
(just the arguments get ignored)

Reply:
It will work on ETH but the asserts would be different (and pointless, obviously)

Reply:
the final transfer's behavior would change